subseas_flex_fcst/maproom.py

```diff
@@ -131,54 +131,75 @@ def write_mapTitle(startDate, leadTime, leadTimeOptions):
     return f'{targetPeriod} {CONFIG["variable"]} Forecast issued {startDate}'
 
 @APP.callback(
-    Output("cdf_graph", "figure"),
-    Output("pdf_graph", "figure"),
-    Output("map", "click_lat_lng"),
     Output("layers_group", "children"),
     Output("latInput", "value"),
     Output("lngInput", "value"),
+    Output("map", "click_lat_lng"),
```

Comment:
I did say I would clean up, but it turns out that it seems convenient to return click_lat_lng to None, so that the function can easily know if it's getting new info from a click, or from a submitted button.

Reply:
It was my understanding that click_lat_lng would always be None if the callback wasn't triggered by a map click. Was I wrong about that?

Reply:
Thank you for correcting my misunderstanding.

It seems strange to have click_lat_lng as both an input and an output--I expected that dash would complain about a circular dependency. But apparently it works ok.

That said, I did some reading and it looks like the way you're supposed to figure out which control triggered the callback is by using `dash.ctx.triggered_id` ([doc](https://dash.plotly.com/dash-html-components/button#determining-which-button-changed-with-dash.ctx)). I tried printing the value of that variable inside `pick_location` and it prints either `None` (on the initial load), `map`, or `submitLatLng`, depending on which control I used. If you use that then you won't need to set click_lat_lng to None.